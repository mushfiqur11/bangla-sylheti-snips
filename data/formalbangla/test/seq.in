ব্যাটারি পার্কে যাওয়ার সময় কিছু পরামর্শ আছে?
এই সপ্তাহে স্মলজ জ্যাজ ক্লাবে কি হচ্ছে?
আমাকে গ্র্যান্ড সেন্ট্রাল থেকে ব্রুকলিন ব্রিজ পর্যন্ত ট্রানজিট নির্দেশনা দাও
৩ দিনের আবহাওয়া কেমন?
শুক্রবার রাতে আমার এয়ারবিএনবি'র সবচেয়ে দ্রুত ভ্রমণ পথ দেখাও
আজ রাতের জন্য টাপাস আর ওয়াইন পরিবেশন করা রেস্টুরেন্টে ৩ জনের জন্য একটা টেবিল বুক করুন
পেন স্টেশনের কাছে স্টেক খেতে দুপুরের খাবারের জন্য একটি টেবিল প্রয়োজন
এই সপ্তাহ শেষে আমার এয়ারবিএনবি আবহাওয়া কেমন হবে?
আজ রাতে গ্যালিতে ৬ জনের জন্য একটা টেবিল বুক করো
মেরী জেন এর সাথে আমার ইটিএ শেয়ার করুন
পানসি এ রাত ৮টায় ৮ জনের জন্য একটি টেবিল বুক করুন
আমার লন্ডন এয়ারবিএনবি'র কাছের সেরা জাদুঘরগুলো দেখাও
পের সে তে এখন কত লোক?
আমরা ৪ জনের একটি দল এবং আমরা সূর্যাস্তের জন্য সেভেন হিলসে একটি টেবিল বুক করতে চাই
বাইরে কি ঠাণ্ডা?
কাল কি আমার সারা দিনের অনুষ্ঠানের কাছে বৃষ্টি হবে?
আমি স্কুলে না যাওয়া পর্যন্ত আমার অবস্থান মাকে জানাও
এই শুক্রবার রাত ৮টার দিকে ডেলমনিকোতে ৪ এর জন্য একটা টেবিল বুক করো।
আমার ফেসবুক ইভেন্টে কি মেঘ থাকবে?
আমি এখনই একটা ট্যাক্সি চাই
আগামীকাল সকালে আবহাওয়া কেমন হবে?
আমাকে বস্টন এলাকার সবচেয়ে বড় ওয়ালমার্টের ঠিকানা দাও
আমাকে কাছাকাছি ভেজি রেস্টুরেন্টগুলো দেখান
আমাকে মন্ড্রিয়ান সোহোর ছবি দেখাও
ম্যাডিসন স্কয়ার গার্ডেনে আজ রাতে কে খেলছে?
সবসময় লতিফের সাথে আমার অবস্থান শেয়ার কর
আমার বাবা-মায়ের আবহাওয়া কেমন?
আমাকে স্টেট গ্রিল আর বারের ঠিকানা দাও
আমার অবস্থান আমার স্বামীকে পাঠাও
আমি ব্লু নোটে কনসার্টের টিকেট কোথা থেকে পেতে পারি?
আমি কিভাবে মোনতাউক-এ যাবো টোল এড়ানোর জন্য?
আমি যেখানে জনের সাথে দেখা করছি সেখানে কি ওয়াইফাই আছে?
স্নিপস দলের সাথে আমার ইটিএ শেয়ার করুন
