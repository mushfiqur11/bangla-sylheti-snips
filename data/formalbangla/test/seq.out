B-place I-place O O O O O O
O O B-place I-place I-place I-place I-place O O O O
O B-origin I-origin O B-destination I-destination I-destination I-destination O O O O O O O
O O O O O
O O B-destination I-destination I-destination I-destination I-destination I-destination I-destination O O O O O
O O O O O O O O O O B-restaurant B-partySize O O O O O O
B-restaurant I-restaurant O O O O O O O O O O
O O O B-location I-location I-location I-location I-location O O O O
O O B-restaurant I-restaurant B-partySize O O O O O O
B-contact I-contact I-contact O O O O O O O O
B-restaurant I-restaurant O O B-partySize O O O O O O O O
B-address I-address I-address I-address I-address I-address I-address I-address O O B-category I-category O
B-place I-place O O O O O
O B-partySize O O O O O O O O O B-restaurant I-restaurant I-restaurant I-restaurant O O O O O
B-location O O O
O O B-location I-location I-location I-location O O O O
O O O O O O O B-contact O O
O O O O O O B-restaurant I-restaurant I-restaurant I-restaurant B-partySize O O O O O O O
O O O O O O O O
O O O B-rideOptionName O
O O O O O O
O B-place I-place I-place O O O O O O O
O O B-specificItemSold I-specificItemSold B-category I-category O
O B-place I-place I-place I-place I-place O O
B-place I-place I-place I-place I-place I-place I-place O O O O O
O B-contact I-contact O O O O O
O O O O O O O
O O O O O O O O
O O B-contact I-contact O O
O B-place I-place I-place I-place I-place O O O O O O
O O B-destination I-destination I-destination I-destination I-destination O O O O O O O O
O O O O O O O O O O O O
B-contact I-contact I-contact I-contact O O O O O O O
